moc_openstack/manifests/ssl/temp_glance_fix.pp

```diff
@@ -0,0 +1,14 @@
+class moc_openstack::ssl::temp_glance_fix (
+  $file   = "/usr/lib/python2.7/site-packages/glanceclient/common/https.py",
+  $source = "puppet:///modules/quickstack/https.py",
+) {
+
```

Comment:
Updated glanceclient caused different other issues. I used "pip install python-glanceclient==0.18 --upgrade" or some other version > 0.17, but it was causing various other dependency issues. Hence, left it as it is.